minecraft-server/minecraftd.sh

```diff
@@ -163,12 +162,12 @@ idle_server_daemon() {
 # Start the server if it is not already running
 server_start() {
 	# Start the game server
-	if ${SUDO_CMD} screen -S "${SESSION_NAME}" -Q select . > /dev/null; then
-		echo "A screen ${SESSION_NAME} session is already running. Please close it first."
+	if ${SUDO_CMD} tmux -L "${SESSION_NAME}" select-window -t0:0 2> /dev/null; then
+		echo "A tmux ${SESSION_NAME} session is already running. Please close it first."
 	else
 		echo -en "Starting server..."
-		${SUDO_CMD} screen -dmS "${SESSION_NAME}" /bin/bash -c "cd '${SERVER_ROOT}'; ${SERVER_START_CMD}"
-		${SUDO_CMD} screen -S "${SESSION_NAME}" -X logfile "${GAME_COMMAND_DUMP}"
+		${SUDO_CMD} mkfifo "${GAME_COMMAND_DUMP}"
```

Comment:
To recover from unexpected terminations, I would add `${SUDO_CMD} rm -f ${GAME_COMMAND_DUMP}` before you create the fifo.

Reply:
Oh, this is another thing I had a question about. Originally the output was redirected to a file and that file was read and deleted. I replace that with a named pipe because it seemed like a good fit for that processing loop but with tmux the output could be captured in some other way. If you aren't worried about keeping screen compatibility, do you think storing that output in a bash variable would be better?

Reply:
hmm, I was probably thinking too much about GNU screen here. Putting it into a bash variable sounds like a way better approach.

Reply:
Actually, now that I've read into what the pipe-pane command does, I'm not sure if it's possible. Probably best to stick to the named pipe approach.